pipeline-stage-tags-metadata/pom.xml

```diff
@@ -36,6 +36,12 @@
   <description>Library plugin for Pipeline stage tag metadata</description>
   <url>https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Model+Definition+Plugin</url>
 
+  <scm>
```

Comment:
@MarkEWaite Aren't these inherited?

Reply:
They are inherited but the inherited values are wrong.  The effective pom (from `mvn help_effective-pom`) in the nested modules include an incorrect value for `scm.url`.  If the parent repository is https://github.com/pipeline with a module named my-module then the module scm.url will be https://github.com/pipeline/my-module, even though it should be https://github.com/pipeline

Reply:
😭